src/dnc/azext_dnc/generated/custom.py

```diff
@@ -0,0 +1,171 @@
+    parameters['identity']['type'] = type_
+    parameters['properties'] = {}
+    parameters['properties']['orchestrator_app_id'] = orchestrator_app_id
+    parameters['properties']['orchestrator_tenant_id'] = orchestrator_tenant_id
+    parameters['properties']['cluster_root_ca'] = cluster_root_ca
+    parameters['properties']['api_server_endpoint'] = api_server_endpoint
+    parameters['properties']['private_link_resource_id'] = private_link_resource_id
+    parameters['properties']['controller_details'] = {}
+    parameters['properties']['controller_details']['id'] = id_
+    return client.begin_create(resource_group_name=resource_group_name,
+                               resource_name=resource_name,
+                               parameters=parameters)
+
+
+def dnc_orchestrator_instance_service_delete(client,
+                                             resource_group_name,
+                                             resource_name):
+    return client.begin_delete(resource_group_name=resource_group_name,
+                               resource_name=resource_name)
+
+
+def dnc_orchestrator_instance_service_patch(client,
+                                            resource_group_name,
+                                            resource_name,
+                                            tags=None):
+    parameters = {}
+    parameters['tags'] = tags
+    return client.patch(resource_group_name=resource_group_name,
+                        resource_name=resource_name,
+                        parameters=parameters)
+
+
+def dnc_orchestrator_instance_service_show_detail(client,
+                                                  resource_group_name,
+                                                  resource_name):
+    return client.get_details(resource_group_name=resource_group_name,
+                              resource_name=resource_name)
+
+
+def dnc_delegated_subnet_service_list(client,
+                                      resource_group_name=None):
+    if resource_group_name:
+        return client.list_by_resource_group(resource_group_name=resource_group_name)
+    return client.list_by_subscription()
+
+
+def dnc_delegated_subnet_service_delete(client,
+                                        resource_group_name,
+                                        resource_name,
+                                        force_delete=None):
+    return client.begin_delete_details(resource_group_name=resource_group_name,
+                                       resource_name=resource_name,
+                                       force_delete=force_delete)
+
+
+def dnc_delegated_subnet_service_patch_detail(client,
+                                              resource_group_name,
+                                              resource_name,
+                                              tags=None):
+    parameters = {}
+    parameters['tags'] = tags
+    return client.begin_patch_details(resource_group_name=resource_group_name,
```

Comment:
Patch method can only update limited parameters. So CLI update command usually use put method to support more parameters' modification.

Reply:
You can follow the example below to manually overwrite custom function

https://github.com/Azure/azure-cli-extensions/blob/19a0ab7bf872ffd457a1e6f54a82b2607e37c4bc/src/monitor-control-service/azext_amcs/manual/custom.py#L80-L117

Reply:
I hide the patch method because our RP should not support it